### DISCUSSION TITLE
Can't downgrade bindgen from .88 to .87

### DISCUSSION THREAD

**Discussion by cybersoulK** [wasm-bindgen/wasm-bindgen#3700] [Q&A]
I am trying to troubleshoot an performance issue, i want to downgrade bindgen to 87, but it's not letting let me.
cargo tree -i reveals all of my crates use the 87 and not 88.

I also tried the reccomendation of installing wasm-server-runner. what can i possibly do?

**Comment by daxpedda** [comment]
What do you mean exactly with "it's not letting me"? What did you try?
You can simply specify a certain version with `wasm-bindgen = "0.2.87"`.
Or you can use `cargo update -p wasm-bindgen --precise 0.2.87`.

**Reply by cybersoulK** [reply]
all of my wasm-bindgen, web-sys, js-sys, futures are the lower version,
it's wasm-bindgen = "0.2.87". i don't have any "0.2.88".

Yes i tried cargo update -p wasm-bindgen --precise 0.2.87.
nothing changed. cargo clean 5 times, removed Cargo.lock, and nothing. that's why i created the issue

**Reply by daxpedda** [reply]
Could you try `wasm-bindgen = "=0.2.87"`?

**Reply by cybersoulK** [reply]
also did, but it didn't even pass the indexing phase

**Reply by daxpedda** [reply]
What is the "indexing phase"? I assume you are looking at rust-analyzer, you should probably stop it until you resolve it, it might be resetting the `Cargo.lock` file for you.

Whats the error message when you just use `cargo build`?

**Reply by cybersoulK** [reply]
i was having a hard time downgrading the Cargo.lock to 0.2.87 until i tried =0.2.87, then i had indexing issue. i removed the =0.2.87 to just 0.2.87, and the cargo.lock remained 0.2.87, but it still says my project has 0.2.88

**Reply by daxpedda** [reply]
> [..], but it still says my project has 0.2.88

Who is "it"?

**Reply by cybersoulK** [reply]
let me revert all of the dependencies back, and give you more detailed error messages, and i will disable RA and see if it solves it

**Reply by cybersoulK** [reply]
Ok, it seems like i had another project compiling in the build script that i didn't think of, and that one also got upgraded.
The = before the version number reverts every other dependency in the workspace, now it works. thank you so much! and sorry of the fuss
